graph.py

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/python3
+import numpy as np
+import matplotlib.pyplot as plt
+import matplotlib.dates as dates
+
+
+# Converts date into readable format.
+def date2num(fmt, encoding='utf-8'):
+    str_converter = dates.strpdate2num(fmt)
+
+    def converter(b):
+        s = b.decode(encoding)
+        return str_converter(s)
+    return converter
+
+
+# Plots a graph based on data provided by "stats.csv".
+# If the dates on "stats.csv" aren't in chronological order, the plot will "zig-zag".
+def graph():
+    date, servers, players = np.loadtxt('stats.csv',
+                                        delimiter=',',
+                                        unpack=True,
+                                        converters={0: date2num('%Y-%m-%d %H:%M')})
```

Comment:
TES3MP Server Stats currently uses Year/Day/Month instead of Year/Month/Day.

Reply:
Grim's main script had the format in Year/Month/Day, went ahead and changed it.